### DISCUSSION TITLE
[Feature Request] Possible future C support

### DISCUSSION THREAD

**Discussion by zkkv** [cabinpkg/cabin#1161] [Ideas]
This tool is essentially what I was looking for: a hassle-free build system akin to cargo with a simple TOML config file, because C and C++ do not have a good build system, unfortunately. I just wish there was C support. I don't know how difficult it is to implement that from a technical point but that would be simply awesome!

Do you plan to add support for C?

**Comment by ken-matsui** [comment]
https://github.com/orgs/poac-dev/discussions/944

Yes, it's definitely planned for the future, but for now, I'm focusing on C++.

**Comment by ken-matsui** [comment]
C is now supported
